app/views/catarse_bootstrap/layouts/_new_zendesk.html.slim

```diff
@@ -1,4 +1,4 @@
-script src="//assets.zendesk.com/external/zenbox/v2.6/zenbox.js" type="text/javascript"
+<script async src="//assets.zendesk.com/external/zenbox/v2.6/zenbox.js" type="text/javascript"></script>
```

Comment:
I could't find a way to set the async attribute with slim. The closest would be setting async=""